typewise_alert.py

```diff
@@ -1,46 +1,33 @@
+from collections import defaultdict 
 
 def infer_breach(value, lowerLimit, upperLimit):
-  if value < lowerLimit:
-    return 'TOO_LOW'
-  if value > upperLimit:
-    return 'TOO_HIGH'
-  return 'NORMAL'
-
+    if value < lowerLimit:
+        return 'TOO_LOW'
+    elif value > upperLimit:
+        return 'TOO_HIGH'
+    return 'NORMAL'
 
 def classify_temperature_breach(coolingType, temperatureInC):
-  lowerLimit = 0
-  upperLimit = 0
-  if coolingType == 'PASSIVE_COOLING':
-    lowerLimit = 0
-    upperLimit = 35
-  elif coolingType == 'HI_ACTIVE_COOLING':
-    lowerLimit = 0
-    upperLimit = 45
-  elif coolingType == 'MED_ACTIVE_COOLING':
-    lowerLimit = 0
-    upperLimit = 40
-  return infer_breach(temperatureInC, lowerLimit, upperLimit)
+    coolingTypes = ['PASSIVE_COOLING', 'HI_ACTIVE_COOLING', 'MED_ACTIVE_COOLING']
+    upperLimit = [35, 45, 40]
```

Comment:
An assumption is made here that lower limit is all zero. I do not see any test cases to safeguard this assumption